### ISSUE TITLE
Support hiding UI elements in the Excalidraw embed package

### ISSUE THREAD

**Issue by gaearon** [excalidraw/excalidraw#3012]
This is concerning the packaged version.

I want to constrain what the user can do. For example, I want to remove a few drawing tools except the most basic ones, hide the button that lets me customize styles, and so on. The end result I want is this:

<img width="534" alt="Screenshot 2021-02-12 at 03 37 54" src="https://user-images.githubusercontent.com/810438/107727678-baa9d380-6ce3-11eb-99af-273f0ee83ddd.png">

I did this with some gnarly css but it feels pretty fragile:

```js
.Island .Stack .Shape:nth-child(3),
.Island .Stack .Shape:nth-child(6),
.Island .Stack .Shape:nth-child(9),
.ToolIcon__lock,
.App-toolbar-content button[aria-label="Edit"],
.App-toolbar-content button[aria-label="Duplicate"] {
  display: none !important;
}
.ExcalidrawReadonly .Island {
  display: none !important;
}
```

If I could somehow customize which UI elements are hidden through props, that would have been much better.

**Comment by ad1992** [comment]
yep, we should allow the host to customize the UI. Similar feature request https://github.com/excalidraw/excalidraw/issues/2513.

**Comment by ad1992** [comment]
We can have 3 props `canvasActions`, `selectedShapeActions` and `shapes` each prop can have an attribute for each item with value `true`/ `false ` as shown 👇. By default, all will be true. Passing `false` or not passing will be treated the same.
```
canvasActions: { save: true, saveAs: true, delete: false, export: false}
shapes: { selection: true, square: true, freeDraw: true }
selectedShapeActions: {stroke: true, backrground: true, fontSize: true, fontFamily: true }
```

Thoughts ? cc @dwelle

**Comment by dwelle** [comment]
Sound reasonable.

It'll also require modifying keyboard shortcut handlers, contextMenu, drag&drop etc., so may become fairly involved. Ideally we shouldn't pepper conditionals and ternaries everywhere coz the codebase is getting too unreadable as it is. We'll figure it in the PR.

Also, let's not allow to disable `selection` as that would impact too much internal code (apart from not making much sense), although I'm sure you listed it above just as an example.

**Comment by ad1992** [comment]
> It'll also require modifying keyboard shortcut handlers, contextMenu, drag&drop etc., so may become fairly involved. Ideally we shouldn't pepper conditionals and ternaries everywhere coz the codebase is getting too unreadable as it is. We'll figure it out in the PR.

Yep we need not push/release all at once, all 3 props can go in individual PR's.

> Also, let's not allow to disable selection as that would impact too much internal code (apart from not making much sense), although I'm sure you listed it above just as an example

yep that was just an example. Selection we won't allow the host to disable
